### PR TITLE
Excluding custom modules from back artifacts

### PR DESCRIPTION
**Pull request by davidferlay**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -158,11 +158,13 @@ prepare:back:
   dependencies: []
   artifacts:
     name: "$CI_COMMIT_REF_NAME:$CI_COMMIT_SHA:back"
-    expire_in: 1d
+    expire_in: 12 hrs
```
Comment (andypost) [inline .gitlab-ci.yml:161]: let's configure it with variable to edit in single place, OTOH there should be a setting in UI

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -158,11 +158,13 @@ prepare:back:
...
     paths:
       - vendor/
       - web/
-      - drush/contrib/
+      - drush/
```
Comment (andypost) [inline .gitlab-ci.yml:165]: no reason, because into "contrib" drush extensions will be added

**Review by andypost — CHANGES_REQUESTED** [review]
please change only "exclude" in the PR

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -158,11 +158,13 @@ prepare:back:
...
+    exclude:
+      - web/modules/custom/
```
Comment (andypost) [inline .gitlab-ci.yml:167]: please add `web/themes/contrib` and `web/profiles/contrib`

File: .gitlab-ci.yml (hunk 1 above)
Comment (davidferlay) [reply .gitlab-ci.yml:161]: tried, gitlab ci doesn't support variable for artifacts TTL

**Pull request merged**